inference-engine/tests/functional/plugin/shared/src/low_precision_transformations/convolution_transformation.cpp

```diff
@@ -79,6 +79,11 @@ void ConvolutionTransformation::validateNGraph() {
 
 TEST_P(ConvolutionTransformation, CompareWithRefImpl) {
     Run();
+    if (targetDevice == "CPU") {
```

Comment:
Please, remove this code from shared lib!

Reply:
Thanks! Done